### ISSUE TITLE
Timeout Page Requires the User to Close 

### ISSUE THREAD

**Issue by eStar386** [CMSgov/eAPD#3121]
Environment: Dev/PR:
After system session timeout based on inactivity, the clickable modal persists.  
![Screen Shot 2021-05-11 at 9.53.48 AM.png](https://images.zenhubusercontent.com/6019ab5a4e513ffb21b40c96/deaa82a4-5a0d-4839-86b2-662c53796a9d)

Environment: Dev/PR and Staging 
Displays incorrect logout message when the user session times out. 
![logoutMessage.png](https://images.zenhubusercontent.com/6019ab5a4e513ffb21b40c96/f8406098-1601-4544-81fd-1b0bd4c034a6)

### This task is done when...
 - [ ] On session timeout, a system and user friendly message appears stating the system logged out user for security purposes or after a period of inactivity.

**Comment by beparticular** [comment]
The time out modal is being updated in #2926. Putting this into the backlog until that update is merged in order to determine what the best course of action, if any, should be based on the updated design. 

When it's time, let's note the intended behaviors vs what is actually happening on the time out modal: 
- When the system times out, does it always persist or disappear after a certain time? 
- When user clicks on "Stay signed in" do they always stay signed in, or do they get logged out after a certain amount of time has passed, even if they say "stay signed in". 
- When user clicks "sign out" they should always be signed out and taken to the log in page. Should they have an error other than "You have securely logged out?" 
- When user clicks outside of the modal what happens?

**Comment by beparticular** [comment]
There are two bugs in the upcoming sprint that might fix this #3126 and 3137.

**Comment by beparticular** [comment]
There will be an additional ticket coming to audit this and add additional time to the timeout. Once those are done, this should be solved.

**Comment by beparticular** [comment]
This has been resolved with the other fixes listed above.
